Lib/tkinter/theme-change.py

```diff
@@ -0,0 +1,47 @@
+import tkinter as tk
+
+# create window
+window = tk.Tk()
+window.geometry('500x500')
+window['bg'] = '#cce'
+window.resizable(height=False, width=False)
+window.wm_attributes('-alpha', 0.95)
+# label
```

Comment:
Creating the non-resizable window.

Reply:
Sorry, I meant code comments :sweat:. It may be worth holding off on these changes, since the issue on bugs.python.org doesn't seem to have support from Serhiy (the core who would probably be the one to merge it)